docs/setup-guide/edge_cluster_copy.md

```diff
@@ -0,0 +1,91 @@
+#	File or Folder Copy Process Between Edge cluster(For Reference)
```

Comment:
rename this file to something like copy_cert_between_edge_cluster.md.

"edge_cluster_copy" is unclear.